task3.0/index.html

```diff
@@ -0,0 +1,405 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=, initial-scale=1.0">
+	<title>Find Me</title>
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Ravi+Prakash&family=Sarala:wght@400;700&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="css/styles.css" />
+</head>
+<body>
+	<div class="wrapper">
+	<header id="header" class="header">
+
+		<div class="logo">
+			<img src="img/logo.svg" align="left" vspace="11" hspace="11" alt="Find Me"> 
+
+		</div>	
+		<div class="all">
+			
+			<div class="tab" >Поможем найти <br>услугу/товар со скидкой</div>
+
+			<div class="bt">
+
+				<p class="p01">Karpenko_20</p>
+				<button type="button2" class="bt2">Добавить объявление</button><div class="img2">
+			<img src="img/icons.png"  alt="Find Me"> </div>
+         </div></div>
+	</header> </div>
+<div class="wrapper1">
+	<div class="page">
+		<div class="filtr">
+			<div class="catalog_sorter">
+				<span>Сортировать по:</span>
+				<a class="sort-item sort_select" id="vend" data-value="vend" href="#">
+					<span>вендорам</span>
+				</a> |
+				<a class="sort-item sort_select" id="day" data-value="day" href="#">
+					<span>дате</span>
+				</a>|
+				<a class="sort-item sort_select" id="hash" data-value="hash" href="#">
+					<span>хештегам</span>
+				</a>
+			</div>
+		</div>
+			<div class="product">
+				<div class="block1">
+		<div class="product-item">
+			<p class="product-title">Книга "Шестерка воронов"</p>
+				<div class="rating">
+					<span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span> 
+				</div><span>#книга</span>
+			<div class="img1">
+			<img src="img/product01.png" alt="product01"></div>
+			<div class="namesale">
+			<div class="name">
+			<p class="nder">OZ</p>
+			<p class="www"><a href="https://oz.by/books/more10613272.html?sbtoken=cec7d4247a6de901380ab59772da5202">Ссылка на товар</a></p>
+		</div>
+		<div class="sale">
+			<p class="product-sale">-30%</p>
+			<p class="before">до 30.03.21</p>
+		</div>
+		</div>
+		<div class="aboutcom">
+			<p class="about">Столичный город Кеттердам - центр международной торговли, место, где проворачивают свои дела крупные бизнесмены и преступные группировки. У Каза Бреккера, юного пижона, в Кеттердаме заслуженная репутация криминального гения, который может все.</p>
+			<div class="comment">
+				<div class="com1">
```

Comment:
You don't need to have different classes for each comment, just use the same class `comment`